runfolder/services.py

```diff
@@ -25,6 +26,10 @@ def __init__(self, host, path, state):
         self.path = path
         self.state = state
         self.service_version = version
+        if metadata:
```

Comment:
Small thing, but the typical python way of doing this as far as I've seen is:
```
if no metadata:
    self.metadata = {}
```

Reply:
I realized that the `get_metadata` function will return an empty dictionary if no values are found, so I'll remove this check completely.